.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
wow, you are fast! I was thinking on do the same :-) 

Anyways, I was thinking to align this template with flow and components somehow 
https://raw.githubusercontent.com/vaadin/flow/master/.github/ISSUE_TEMPLATE.md
https://github.com/vaadin/vaadin-grid/issues/new

Another options are:
option 1. in this template we should encourage ppl to open the issue in the appropriate repo if they can, e.g flow or appropriate component
option 2. have more templates for generic issues, component issues, flow issues, then we can move those to the appropriate repo

Reply:
that is probably good. i think the main content of those templates is quite similar. (you are free to change the contents here too.)

we can mention those to encourage ppl. but TBH, normally only the new users will use our repo for bugs, experienced users will find the correct one.